code/obj/item/kitchen.dm

```diff
@@ -511,6 +511,25 @@ TRAYS
 		return "[food_desc]"
 
 	attackby(obj/item/W as obj, mob/user as mob)
+		if (istype(W, /obj/item/plate) && !istype(W, /obj/item/plate/tray))
+			qdel(W)
+			src.set_loc(user)
+			user.put_in_hand_or_drop(new /obj/item/platestack)
+			user.visible_message("<b>[user]</b> adds a plate to the stack.","You add a plate to the stack.")
+			qdel(src)
+			return
+		else if (istype(W, /obj/item/platestack))
+			var/obj/item/platestack/stack = W
+			if(stack.platenum >= 7)
```

Comment:
I know we don't really have a "standard" for it or anything throughout the code-base, so no big deal, but keeping consistent in your own changes is good with regards to stuff like `if (` vs. `if(`.